jetty-core/jetty-io/src/main/java/org/eclipse/jetty/io/ArrayByteBufferPool.java

```diff
@@ -576,6 +627,17 @@ private int evict()
             return getCapacity();
         }
 
+        private Statistics getStatistics()
+        {
+            List<Pool.Entry<RetainableByteBuffer>> entries = getPool().stream().toList();
+            int inUse = (int)entries.stream().filter(Pool.Entry::isInUse).count();
```

Comment:
Converting to a list is needlessly wasteful, you could directly stream the pool here with:
```java
int inUse = (int)getPool().stream().filter(Pool.Entry::isInUse).count();
```